Microsoft.Toolkit.Uwp.UI.Media/Effects/BlendEffect.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Graphics.Canvas.Effects;
+using Microsoft.Toolkit.Uwp.UI.Media.Effects.Interfaces;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
+{
+    /// <summary>
+    /// A blend effect that merges the current pipeline with an input one
+    /// </summary>
+    /// <remarks>This effect maps to the Win2D <see cref="Graphics.Canvas.Effects.BlendEffect"/> effect</remarks>
+    public sealed class BlendEffect : IPipelineEffect
+    {
+        /// <summary>
+        /// Gets or sets the input pipeline to merge with the current instance
+        /// </summary>
+        public IList<IPipelineEffect> Input { get; set; } = new List<IPipelineEffect>();
+
+        /// <summary>
+        /// Gets or sets the blending mode to use (the default mode is <see cref="BlendEffectMode.Multiply"/>)
+        /// </summary>
+        public BlendEffectMode Mode { get; set; }
```

Comment:
We should use the ImageBlendMode enum proxy as it removes the property not supported by the composition engine. (then just do an int/enum cast to convert when needed?)

Reply:
Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3112/commits/65a2b14ba8b4fd2bbd602ec900eaf6842b08f92b.